src/tests/JIT/Regression/JitBlue/Runtime_90323/Runtime_90323.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+public class Runtime_90323
+{
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private static float ConvertToSingle(long value) => (float)value;
+
+    // 32-bit currently performs a 2-step conversion which causes a different result to be produced
```

Comment:
"currently" is a problematic word for a long-lived test.

Reply:
The current plan of record is to fix the places that are doing incorrect conversions early in .NET 9.

I can certainly remove "currently" in the interim if that helps?